### PR TITLE
IIB 6.6.1 release

### PR DESCRIPTION
**Pull request by lipoja**
- Add recursive-related-bundles endpoint
- Adding ability to create single-active-consumer queues
- Add static types and mypy checks
- Append overwrite index token to current docker config
- Fixed config loading

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## 6.6.0
+## 6.6.1
 - Add recursive-related-bundles endpoint
```
Comment (yashvardhannanavati) [inline CHANGELOG.md:10]: Should we have two releases in the release notes? 6.6.1 only having the bug fix and the rest in 6.6.0

**Review by yashvardhannanavati — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (chandwanitulsi) [reply CHANGELOG.md:10]: Only 6.6.1 looks okay to me since 6.6.0 was not released, but no strong objections to having both releases here

**Review by chandwanitulsi — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lipoja) [reply CHANGELOG.md:10]: GitHub release of 6.6.0 was never done - we have only  v6.6.0 tag present...

Comment (yashvardhannanavati) [reply CHANGELOG.md:10]: Ok, good to go with 6.6.1

**Review by yashvardhannanavati — APPROVED** [review]

**Review by yashvardhannanavati — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lipoja) [reply CHANGELOG.md:10]: And I hope that v6.6.1 will be working I am planning to do the GitHub release - which will create official IIB images and release

**Pull request merged**